pkg/mempool/simplemempool/parts/computeids/computeids.go

```diff
@@ -43,11 +42,7 @@ func IncludeComputationOfTransactionAndBatchIDs(
 
 	mppbdsl.UponRequestBatchID(m, func(txIDs []tt.TxID, origin *mppbtypes.RequestBatchIDOrigin) error {
 		data := make([][]byte, len(txIDs))
-		copy(data, txIDs)
-
-		if len(txIDs) == 0 {
-			mppbdsl.BatchIDResponse(m, origin.Module, emptybatchid.EmptyBatchID(), origin)
-		}
+		copy(data, txIDs) // TODO: Why do we copy the txIDs here? Is it because the hasher, technically, doesn't promise be read-only?
```

Comment:
Not actually needed at the moment because of the type aliases so they are both the same type really. Once we fix #356 and define types (instead of using aliases), then the copy would make sense. Can be removed for now.

Reply:
Thanks, I'll remove it then.